src/banana/data/genpdf/project.py

```diff
@@ -0,0 +1,134 @@
+import copy
```

Comment:
Maybe the module names is not the best choice...

Something like `flavors.py` might be a bit more explicative, but I'll keep thinking on a better candidate.